firmware/quadruna/BMS/src/tasks.c

```diff
@@ -259,40 +77,16 @@ void tasks_init(void)
     hw_adcs_chipsInit();
 
     hw_hardFaultHandler_init();
-    hw_can_init(&can);
-    hw_sd_init(&sd);
     hw_crc_init(&hcrc);
     hw_watchdog_init(hw_watchdogConfig_refresh, hw_watchdogConfig_timeoutCallback);
 
-    io_canTx_init(io_jsoncan_pushTxMsgToQueue);
-    io_canTx_enableMode(CAN_MODE_DEFAULT, true);
-    io_can_init(&can_config);
-    io_bmsShdn_init(&bms_shdn_pin_config);
     io_tractiveSystem_init(&ts_config);
-    io_thermistors_init(&thermistors_config);
     io_ltc6813Shared_init(&ltc6813_spi);
-    io_airs_init(&airs_config);
     io_imd_init(&imd_pwm_input_config);
-    io_chimera_init(&debug_uart, GpioNetName_bms_net_name_tag, AdcNetName_bms_net_name_tag, &n_chimera_pin);
+    io_chimera_init(GpioNetName_bms_net_name_tag, AdcNetName_bms_net_name_tag);
     io_sdGpio_init(&sd_gpio);
-    io_bspdTest_init(&bspd_test_config);
-
-    app_canTx_init();
-    app_canRx_init();
-
-    app_accumulator_init();
-    app_tractiveSystem_init();
-
-    // Re-enable if auxiliary thermistors installed
-    // app_thermistors_init();
-
-    app_soc_init();
-    app_globals_init(&globals_config);
-    app_stateMachine_init(app_initState_get());
 
-    // broadcast commit info
-    app_canTx_BMS_Hash_set(GIT_COMMIT_HASH);
-    app_canTx_BMS_Clean_set(GIT_COMMIT_CLEAN);
+    app_globals_init();
```

Comment:
this probably needs to be done in jobs right

Reply:
yes, but I think globals needs to be gone just in general

Reply:
Only VC is really up to new spec, all the other boards I just changed so that they still compile.

Reply:
yeah thats fair. i think there was just a couple cases where I need to share data between multiple states which is the only reason globals ever exited. probably better ways of doing this

Reply:
yeah, I think making a module which is more descriptive but has similar functionality at the end of the day is always better